File: src/main/java/com/github/kory33/signvote/vote/VoteScore.java
package com.github.kory33.signvote.vote;

import lombok.EqualsAndHashCode;
import lombok.RequiredArgsConstructor;

/**
 * Represents a vote's score
 */

@RequiredArgsConstructor
@EqualsAndHashCode
public final class VoteScore {
    private final int score;

    public int toInt() {
        return this.score;
    }
}

File: src/main/kotlin/com/github/kory33/signvote/vote/VoteScore.kt
package com.github.kory33.signvote.vote

/**
 * Represents a vote's score
 */

data class VoteScore
constructor(private val score: Int) {

    fun toInt(): Int {
        return this.score
    }

    override fun toString(): String {
        return this.score.toString()
    }
}
